docs/zarr_sink_example.ipynb

```diff
@@ -0,0 +1,545 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9190dbbd",
+   "metadata": {},
+   "source": [
+    "# Using the Zarr Tile Sink\n",
+    "\n",
+    "The `ZarrFileTileSource` class has file-writing capabilities; a user may start with an empty Zarr store, add data tiles, then save the data to a file of any format. \n",
```

Comment:
This is along the lines of how much do we need to explain about external packages.  Someone new to zarr won't know what store is and doesn't need to know to use our library.  And, although we add data via `addTile`, we really are adding arbitrary regions, so perhaps a better wording would be "an empty image can be created, image data can be added as tiles or arbitrary regions, and the image can be saved to a file in any of several formats".